tests/commands/gen-js/__snapshots__/index.d.ts

```diff
@@ -176,7 +176,7 @@ export interface ExampleEvent {
   /**
    * Required object (empty) property
    */
-  "required object (empty)": { [key: string]: any };
```

Comment:
TODO: just noticed this bug. We should be allowing `{ [key: string]: any }` for properties (just not for the top-level event properties object). Otherwise you can't embed objects with an unknown type, like a JS error object.

Reply:
Fixed with: https://github.com/segmentio/typewriter/pull/56/commits/95abfbd73142810c5d04b87d19a0e3384ad0b189